client/js/contextMenuFactory.js

```diff
@@ -153,6 +153,22 @@ function addFocusItem() {
 	});
 }
 
+function addEditNetworkItem() {
+	function edit(itemData) {
+		socket.emit("network:get", itemData);
+		$("#connect").text("Loading");
```

Comment:
Is this loading text legacy from a previous attempt or something? I couldn't manage to trigger it, even when emulating slow network and even when simulating lost connection.

However, I do notice a flash when clicking on "Edit this network", for a very short window (100ms?) the fields are filled with the connect defaults, then change to the actual values. I'm wondering if we can keep them empty until they're filled.